tests/conftest.py

```diff
@@ -171,6 +196,15 @@ def ptfhost(testbed_devices):
     return testbed_devices["ptf"]
 
 
+@pytest.fixture(scope="module")
```

Comment:
Please double check, looks like it is not needed

Reply:
Fixed.